test/Microsoft.AspNet.Mvc.IntegrationTests/ValidationIntegrationTests.cs

```diff
@@ -1000,7 +1000,7 @@ private class Order10
             var model = Assert.IsType<List<Order10>>(modelBindingResult.Model);
             Assert.Empty(model);
 
-            //Assert.Equal(0, modelState.Count); // This fails due to #2446
+            //Assert.Equal(0, modelState.Count);
```

Comment:
I did a blind find replace will take a look if this can be enabled. BTW the current test is still skipped.